src/gapic-generator-typescript.ts

```diff
@@ -80,7 +80,7 @@ const packageName = argv.packageName as string | undefined;
 const mainServiceName = argv.mainService as string | undefined;
 const template = argv.template as string | undefined;
 const gapicValidatorOut = argv.gapicValidatorOut as string | undefined;
-const validation = argv.validation as string | 'true';
+const validation = argv.validation as string | undefined;
```

Comment:
The modern way to do it is `const validation = (argv.validation as string | undefined) ?? 'true'`, then in the condition just check `... && validation === 'true'` and it will work as expected.

Reply:
Thank you for the advice, done.